app/models/project.rb

```diff
@@ -9,13 +9,14 @@ class Project < ActiveRecord::Base
   has_many :orders, through: :loans
   belongs_to :tenant
 
-  validates :description, length: { in: 0..255, allow_nil: false }, presence: true
+  validates :description, length: { in: 100..255, allow_nil: false },
```

Comment:
Yes, so I put a chore (#87) on the waffle board to discuss whether we want allow the description to be longer than 255 (which requires changing the field from string to text). Sound like you vote yes!